web-ui/src/pages/ReceivedRequestsPage.jsx

```diff
@@ -309,18 +299,16 @@ const ReceivedRequestsPage = () => {
       </Collapse>
       <div className="request-section-header">
         <Typography variant="h5">Submitted Requests</Typography>
-        <IconButton
+        <ExpandMore
+          expand={submittedRequestsExpanded}
           onClick={() =>
             setSubmittedRequestsExpanded(!submittedRequestsExpanded)
           }
+          aria-expanded={submittedRequestsExpanded}
           aria-label="show more"
-          className={
-            submittedRequestsExpanded ? classes.expandOpen : classes.expandClose
-          }
-          size="large"
         >
           <ExpandMoreIcon />
```

Comment:
Why doesn't the `ExpandMore` component supply `ExpandMoreIcon` so it isn't needed in this file?